docker/app/Dockerfile

```diff
@@ -1,8 +1,12 @@
+FROM ghcr.io/uoy-research/rdkit-sqlite:main
+
 # Install tools used when installing / building gems
 # Use a debian ruby image rather than the base lambda one as that expects
 # a lambda entry point
 FROM ruby:3.2-slim
-RUN apt-get update -qq && apt-get install -y build-essential libsqlite3-dev
+
+COPY --from=0 /lib/librdkit* /lib/
+RUN apt-get update -qq && apt-get install -y build-essential libsqlite3-dev libboost-all-dev libfreetype-dev
```

Comment:
Can you double check that all of these packages are still needed following the extension build being moved to the other repo?